packages/react-compass/src/speed-dial/speed-dial.tsx

```diff
@@ -71,12 +72,21 @@ const SpeedDial = React.forwardRef<HTMLDivElement, SpeedDialProps>(
           className={`speed-dial__trigger ${isOpen ? 'open' : ''}`}
           onMouseEnter={handleMouseEnter}
         >
-          <span>{/* <Icon icon={faPlus}></Icon> */}+</span>
+          <span
```

Comment:
Inline style might block the other to override the style, it should be class name or options